apps/hello-world-vuejs/src/App.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <div id="app">
+    <div>Hello {{ user.name }}</div>
+    <div>Your context: {{ JSON.stringify(context) }}</div>
+    <div>Your settings: {{ JSON.stringify(settings) }}</div>
+  </div>
+</template>
+
+<script>
+import mondaySdk from "monday-sdk-js";
+const monday = mondaySdk();
+
+export default {
+  name: "app",
+  data() {
+    return {
+      settings: "",
+      context: "",
+      user: ""
+    };
+  },
+
+  created() {
+    monday.listen("settings", this.getSettings);
+    monday.listen("context", this.getContext);
+    this.fetchUserName();
+  },
+
+  methods: {
+    beforeCreate() {
```

Comment:
i'm not familiar with vue but is beforeCreate like componentwillmount or the constructor i think it should be on the mounted life cycle